_redirects

```diff
@@ -38,4 +38,7 @@ https://developer.aiven.io/* https://docs.aiven.io/:splat 301!
 
 # Renamed/deleted files
 /docs/products/flink/howto/real-time-alerting-solution-cli.html       /docs/products/flink/howto/real-time-alerting-solution.html
-/docs/platform/howto/disable-user-2fa.html                            /docs/platform/howto/user-2fa.html
+/docs/platform/howto/disable-user-2fa.html                            /docs/platform/howto/user-2fa.html                                  301!
```

Comment:
Tried adding the force redirect but didn't resolve the redirect loop - can see in this [PR preview](https://deploy-preview-1322--developer-aiven-io-preview.netlify.app/docs/platform/howto/disable-user-2fa.html). Will open an Netlify support ticket. Similar case for `/docs/tools/terraform/upgrade-provider-v1-v2.html`

Reply:
@lornajane Netlify Support suggested to swap the order of the redirect rules above 
`# Redirect from .index.html to specific page names for landing`

And now the following 2 redirect loop are resolved and redirecting to the correct url.
https://deploy-preview-1322--developer-aiven-io-preview.netlify.app/docs/platform/howto/disable-user-2fa.html
https://deploy-preview-1322--developer-aiven-io-preview.netlify.app/docs/tools/terraform/upgrade-provider-v1-v2.html

Reply:
Oh that sort of makes sense now. Thank you for fixing this!